Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -1015,13 +1018,127 @@ void CGContextClipToRects(CGContextRef context, const CGRect* rects, unsigned co
     CGContextClip(context);
 }
 
+HRESULT __CGContext::ClipToD2DMaskBitmap(ID2D1Bitmap* bitmap, bool shouldInterpolate, CGRect rect) {
```

Comment:
Good idea.

---
In reply to: [92092026](https://github.com/Microsoft/WinObjC/pull/1528#discussion_r92092026) [](ancestors = 92092026)